library/agent/api/Event.ts

```diff
@@ -1,12 +1,15 @@
 import { Kind } from "../Attack";
 import { Source } from "../Source";
 
+type Version = string;
+type PackageName = string;
+
 export type AgentInfo = {
   dryMode: boolean;
   hostname: string;
   version: string;
   library: string;
-  packages: Record<string, string>;
+  packages: Record<PackageName, Version>;
```

Comment:
Are we still going to use this one?

Reply:
https://github.com/AikidoSec/firewall-node/blob/0d31cd93838f809026425ec913cedecd551b7134/library/agent/Agent.ts#L426-L436

Reply:
Let's keep it for now and remove in another PR?